receive.js

```diff
@@ -1,24 +1,44 @@
-"use strict";
-var Q = require("q");
+'use strict';
+const Q = require('q');
 
 exports.process = function (msg, conf) {
-    let msgId = msg.id;
-    console.log("Received new message with id", msgId);
+    const msgId = msg.id;
+    console.log('Received new message with id', msgId);
     console.log(msg);
 
-    if (msg.query && msg.body) {
-        msg.body._query = msg.query;
+    if (msg.body) {
+        if (msg.query) {
+            msg.body._query = msg.query;
```

Comment:
In the task, it wasn't necessary to redefine `msg.body._query`. Why did you decide to do this?

Reply:
I didn't redefine ```msg.body._query```, it still equals ```msg.query```